apps/web-giddh/src/app/material-ledger/ledger.component.html

```diff
@@ -658,6 +658,29 @@ <h3 class="ledger-main-head">
                                                                 class="select2-parent select2-ledger pl-0 pr-0"
                                                             >
                                                                 <div
+                                                                    *ngIf="!txn.showAccountDropdown"
+                                                                    class="w-100"
+                                                                    (click)="
+                                                                        showBankLedgerPopup(txn, item);
+                                                                        txn.showAccountDropdown = true
+                                                                    "
+                                                                >
+                                                                    <span
+                                                                        [ngClass]="{
+                                                                            'account-placeholder yes': true,
```

Comment:
Please check class name
```suggestion
                                                                            'account-placeholder-yes': true,
```